### ISSUE TITLE
Detectron2 - DensePose

### ISSUE THREAD

**Issue by adam-peaston-SC** [StrongResearch/isc-demos#34]
### Source / repo
https://github.com/facebookresearch/detectron2/tree/main/projects/DensePose

### Model description
GeneralisedRCNN

### Dataset
COCO

### Literature benchmark source
[URL]

### Literature benchmark performance
[DESCRIPTION]
[VALUE/S]

### Strong Compute result achieved
[VALUE/S]

### Basic training config (as applicable)
Nodes: [N]
Epochs: [N]
Effective batch size: [N]
Learning rate: [L]
Optimizer: [OPT]

### Logs gist
[URL]
